packages/datadog-plugin-pg/test/index.spec.js

```diff
@@ -609,7 +649,9 @@ describe('Plugin', () => {
 
         it('query should inject _dd.dbm_trace_injected into span', done => {
           agent.assertSomeTraces(traces => {
-            assert.strictEqual(traces[0][0].meta['_dd.dbm_trace_injected'], 'true')
+            assertObjectContains(traces[0][0].meta, {
+              '_dd.dbm_trace_injected': 'true'
+            })
```

Comment:
Nit: This one could be reverted. Maybe there's similar situations in the files above that I missed. Consider validating